### PR TITLE
[4.0]Added lang string for upload file

### PR DESCRIPTION
**Pull request by bahl24**
Signed-off-by: Nitish Bahl <nitishbahl24@gmail.com>

Pull Request for Issue #23947  .

### Summary of Changes
Added lang string for translation in media manager

### Expected result
Lang file added for translation purposes

### Documentation Changes Required
No

### CODE REVIEW

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 42004afb15d119405c525a8da4b38b7c73c9c6df<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23951">issues.joomla.org/tracker/joomla-cms/23951</a>.</sub>

**Comment by infograf768** [comment]
You also have to add the string in
`/administrator/components/com_media/tmpl/media/default_texts.php`

`Text::script('COM_MEDIA_DROP_FILE', true);`

**Comment by bahl24** [comment]
@infograf768 I have added that

**Comment by infograf768** [comment]
Thanks, it will work now. 
It is better to always test your patches on a local site.

NOTE: we will have to alpha order these in another PR as anyway we have to modify/add some new strings in that file.

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 77a22f7db7ab1bd0f12ba17a948b674ad67cd491<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23951">issues.joomla.org/tracker/joomla-cms/23951</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 77a22f7db7ab1bd0f12ba17a948b674ad67cd491<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23951">issues.joomla.org/tracker/joomla-cms/23951</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23951">issues.joomla.org/tracker/joomla-cms/23951</a>.</sub>

**Comment by bahl24** [comment]
@wilsonge Kindly merge this if its ok

**Comment by infograf768** [comment]
Please resolve conflicts. I modified the default_texts.php and this was merged. Therefore the string should be added in alphabetic order in that file AND also in both com_media.ini files (front and backend)

**Comment by bahl24** [comment]
@infograf768 I have made the changes.

File: administrator/components/com_media/tmpl/media/default_texts.php [hunk 1]
```diff
@@ -25,6 +25,10 @@
 Text::script('COM_MEDIA_DECREASE_GRID', true);
 Text::script('COM_MEDIA_DELETE_ERROR', true);
 Text::script('COM_MEDIA_DELETE_SUCCESS', true);
+Text::script('COM_MEDIA_CONFIRM_DELETE_MODEL', true);
```
Comment (Quy) [inline administrator/components/com_media/tmpl/media/default_texts.php]: Please sort in alpha order.

Comment (bahl24) [reply administrator/components/com_media/tmpl/media/default_texts.php]: Sorry, but they are already there in the file. I added them by mistake, @infograf768 has already merged them, I think.

**Comment by infograf768** [comment]
@wilsonge.
Please merge before we have again to to update branch.

**Comment by infograf768** [comment]
@bahl24 
The new string has also to be added in the frontend ini file

**Comment by infograf768** [comment]
I added it. waiting for Checks to pass to merge.

**Comment by infograf768** [comment]
Thanks

**Pull request merged**